php-fpm/php7.3.ini

```diff
@@ -918,12 +918,12 @@ default_socket_timeout = 60
 ; See http://www.php.net/manual/en/snmp.installation.php
 ;extension=php_snmp.dll
 
-;extension=php_soap.dll
+extension=php_soap.dll
 ;extension=php_sockets.dll
 ;extension=php_sqlite3.dll
 ;extension=php_tidy.dll
 ;extension=php_xmlrpc.dll
-;extension=php_xsl.dll
+extension=php_xsl.dll
```

Comment:
@shoxy 
you have added this with a reason `enabled php_xsl extension`
could you remember what was the real reason?
As I don't remember we are using any xsl functions.